src/constants/endpoints.js

```diff
@@ -10,4 +10,7 @@ module.exports = {
 		? process.env.USER_PROFILE_DETAILS_ENDPOINT
 		: 'v1/user/read',
 	VALIDATE_SESSIONS: 'v1/account/validateUserSession',
+	BROWSE_EXISTING_END_POINT: process.env.BROWSE_EXISTING_END_POINT
```

Comment:
@adithyadinesh0412 this will not be via env

Reply:
removed from env